semantic/src/site/globals/site.overrides

```diff
@@ -11,8 +15,112 @@ p strong {
   font-weight: @semiboldWeight;
 }
 
-.ui.basebody.container {
-  background: rgb(248, 248, 248);
+#main-container {
+  padding: 0;
+  position: relative;
+  min-height: 100vh;
+  // Max width is 1027px, from Semantic-UI Container
+}
+
+#user-area {
+  font-family: @userAreaFont;
+  color: @userAreaText;
+  display: flex;
+  position: fixed; // TO-DO: No fixed on mobile
+  background-color: @headersInteractiveLow;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: @headerHeight; // TO-DO: Adjust for mobile
+  justify-content: space-between;
+  //   top: 0,
+  //   zIndex: 1000,
+}
+
+#user-area.ui.container {
+  margin-left: 0px !important;
+  margin-right: 0px !important;
+}
+
+#user-area-left {
+  display: flex;
+  flex-direction: column;
+}
+
+#menu-bar {
+  // border: 1px white solid;
+}
+// Menu bar links
+#user-area a {
+  color: @userAreaText;
+  font-size: 14px;
+}
+
+// Not currently used, but should be appplied to the Link in the menu
+// bar that reflects the current page
+.selected-link {
+  font-weight: bold;
+}
+
+#org-selector {
+  font-size: 27px;
+  letter-spacing: 3.4px;
```

Comment:
I have included one variable for this one (on a later PR), you might want to add and use as part of this too?
https://github.com/openmsupply/application-manager-web-app/blob/f14d65dc8707db2dd4a755cd2c0514bcf1ea37f9/semantic/src/site/globals/site.variables#L76-L79